### DISCUSSION TITLE
我觉得 issue 模板里的描述不够准确

### DISCUSSION THREAD

**Discussion by cssxsh** [DreamVoid/MiraiMC#358] [建议 / Ideas]
@DreamVoid 

其实对于很多项目汇报日志时
能把栈堆列出就已经达到其基本需求了

你这里的 `完整` 对于用户来说其实并不明确

换成 `从启动开始到报错` 之类的时序描述比较好

**Comment by DreamVoid** [comment]
有一点特殊的是，Bukkit搭载mirai远比直接运行mirai复杂，我需要了解Bukkit的版本、是否有其他干扰插件启动的因素（比如复合端、旧版TabooLib的冲突），没有完整的日志就没法正确定位问题。并且能让根本没仔细看日志的再看一遍日志。
